src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -297,14 +297,14 @@ private static void assertGVCFHasOnlyOneSample(final String variantPath, final V
     }
 
     @VisibleForTesting
-    static SortedMap<String, Path> loadSampleNameMapFile(final Path sampleToFileMapPath) {
+    public static Map<String, Path> loadSampleNameMapFile(final Path sampleToFileMapPath) {
```

Comment:
By changing the return value from `SortedMap` back to `Map`, you have made the tool less safe to modify in the future, which we should definitely not be doing at this point. Recommend that you add a method `loadSampleNameMapFileInSortedOrder(sampleNameMapFile)` that is just `new TreeMap<>(loadSampleNameMapFile(IOUtils.getPath(sampleNameMapFile)));`, add a comment that *only* the sorted version should be called directly by the tool, and call `loadSampleNameMapFileInSortedOrder()` above instead of doing an inline sort.

Reply:
done, and I added tests for ordering in both versions